### PR TITLE
feat(simint): add and remove properties from simulators API

### PR DESCRIPTION
**Pull request by bkuzma**

### CODE REVIEW

File: packages/alpha/src/types.ts [hunk 1]
```diff
@@ -116,6 +117,7 @@ export interface SimulatorIntegrationCreate {
 
 export interface SimulatorIntegrationFilter {
   simulatorExternalIds?: CogniteExternalId[];
+  active: boolean;
```
Comment (polomani) [inline packages/alpha/src/types.ts]:
```suggestion
  active?: boolean;
```

Comment (polomani) [reply packages/alpha/src/types.ts]: it's a filter, so optional

Comment (bkuzma) [reply packages/alpha/src/types.ts]: Fixed it 👍

**Review by polomani — APPROVED** [review]
LGTM

**Pull request merged**